ui.tests/test-module/specs/fileinput/fileinput.runtime.spec.js

```diff
@@ -80,7 +80,7 @@ describe("Form with File Input - Basic Tests", () => {
     const checkHTML = (id, state) => {
         const visible = state.visible;
         const passVisibleCheck = `${visible === true ? "" : "not."}be.visible`;
-        const passDisabledAttributeCheck = `${state.enabled === false ? "" : "not."}have.attr`;
+        const passDisabledAttributeCheck = `${state.enabled === false || state.readOnly === true ? "" : "not."}have.attr`;
```

Comment:
As discussed in the scrum, can you please add the same check in all the test specs for all the components ?

Reply:
Sure Rishi, Will be adding the test cases in the next pr which I am working on for tnc component